Ham/DabraLibanosHamGG1/msDesc.xml

```diff
@@ -0,0 +1,1782 @@
+              <locus from="25vb1" to="25vb25"/>
+              <title ref="LIT6335CCR28"/>
+              <incipit>ወሀብኩ፡ አነ፡ ይሥምከ፡ እግዚእ፡ ሥዩመ፡ ኀማሳን፡ ለአባ፡ መጣዕ፡ ዘአሀም፡ ፈዋሴ፡ ዱያን፡ </incipit>
+              <explicit>ቃለ፡ መክብብ፡ ዘይቤ፡ ኵሉ፡ ከንቱ፡ ከንቱ፡ ውእቱ፡ ዘኢገብረ፡ ሠናየ፡ በሕይወቱ። ወበዐጋማሂ፡ አርታዕቱዕኩ፡ ጕልተ። </explicit>
+              <note>[5.12] = CCR 28: (II f. 13vb), according to AB. Conti Rossini comments on pp.
+                213-214: <cit><quote xml:lang="it">Scrittura del secolo XVI: colonna. [...]. Atto non compiuto. Il donatore
+                di schiavi al convento è il capo dello Hamasen, che assume il semplice titolo di
+                sum: nell'atto n.20, che è dell'anno 1328, il capo dello Hamasen è detto awfari, con
+                un vicario (?) chiamato aqansan, ai tempi di re Zara Yaqob aveva invece il titolo di
+                kantiba (J. Perruchon, Les chroniques de Zara Yaqob, p. 48). Riesce quindi
+               <date notAfter="1600"> impossibile precisare la data dell'atto, che in ogni caso non è posteriore al secolo
+                XVI</date></quote>
+                  <bibl><ptr target="bm:ContiRossini1901Evangelodoro"/>
+                    <citedRange unit="page">213–214</citedRange></bibl></cit>. 
+                Roger Schneider noted the following <cit><quote>The text is the second column of the
+                page. The first column, which has 20 lines, has been erased and only faint traces
+                are visible</quote></cit>. </note>
+            </msItem>
+          </msContents>
+          <physDesc>
+            <objectDesc form="Leaf">
+              <supportDesc>
+                <support>
+                  <material key="parchment"></material>
+                </support>
+                <extent>
+                  <locus from="25" to="25"/>
+                </extent>
+              </supportDesc>
+              <layoutDesc>
+                <layout xml:id="layout7" columns="2" writtenLines="25">
+                  <locus from="25" to="25"/>
+                </layout>
+              </layoutDesc>
+            </objectDesc>
+            <handDesc>
+              <handNote xml:id="h23" script="Ethiopic" corresp="#p9_i1">
+                <locus from="25ra1" to="25rb26"/>
+                <seg>CCR dated this hand to the XVII-XVIII century.</seg>
+              </handNote>
+              <handNote xml:id="h24" script="Ethiopic" corresp="#p9_i2">
+                <locus from="25vb1" to="25vb25"/>
+                <seg>CCR dated this hand to the XVI century.</seg>
+              </handNote>
+            </handDesc>
+          </physDesc>
+        </msPart>
+<!--  f.26-->
+        <msPart xml:id="p10">
+          <msIdentifier>
+            <repository ref="INS0346DL"/>
+          </msIdentifier>
+          <msContents>
+            <msItem xml:id="p10_i1">
+              <locus from="26r3" to="26r29"/>
+              <!--              I think this should be LIT6305CCR7, and there should be no second work record
```

Comment:
it is a bit question whether it is one text or two. But I am ready to accept them as one. eshy. you won :-)